connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -135,6 +134,14 @@ public Response putConnectorConfig(final @PathParam("connector") String connecto
         return response.entity(createdInfo.result()).build();
     }
 
+    @POST
+    @Path("/{connector}/restart")
+    public void restartConnector(final @PathParam("connector") String connector) throws Throwable {
```

Comment:
Where is the `forward` query parameter? Without it, I think you will route through the leader, may rebalance, and then forwarding will just continue? And probably even worse if a worker gets a request, gets kicked out of the worker group, and just has stale leader info?